ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -136,6 +143,28 @@ def _llmobs_set_meta_tags_from_chat_model(
                     )
         span.set_tag_str(OUTPUT_MESSAGES, json.dumps(output_messages))
 
+    def _llmobs_set_meta_tags_from_chain(
+        self, span: Span, inputs: Union[str, Dict[str, Any], List[Union[str, Dict[str, Any]]]], input_parameters: dict
+    ) -> None:
+        span.set_tag_str(SPAN_KIND, "workflow")
+
+        if inputs is not None:
+            if isinstance(inputs, dict):
+                input_parameters.update(inputs)
+            elif isinstance(inputs, list):  # batched chain call
+                input_parameters = {}
+                for idx, inp in enumerate(inputs):
+                    if isinstance(inp, dict):
+                        for k, v in inp.items():
+                            input_parameters[f"input.{idx}.{k}"] = v
+                    else:
+                        input_parameters[f"input.{idx}"] = inp
+            elif isinstance(inputs, str):
+                span.set_tag_str(INPUT_VALUE, inputs)
```

Comment:
I thought using `INPUT_VALUE` here for a singular string input to the chain might be cleaner than seeing `{input: { 0: '...' } }` in the UI, but can change it to that if we want 100% consistency